tests/test_cases/suite.sh

```diff
@@ -1,5 +1,6 @@
 mkdir -p automated_tests
 mkdir -p lind_tests
+rm -f foo.txt &> /dev/null
```

Comment:
Forkopen creates a foo.txt and if the test does not succeed, it does not get deleted in the last line. This makes sure the file is deleted.